datalogger/datalogger.proto

```diff
@@ -37,12 +37,14 @@ message DataloggerPayload {
     CanMessage receivedCanMessage = 3;
     CanMessage transmittedCanMessage = 4;
     CanError canError = 5;
-    CanErrorCounter canErrorCount = 6;
-    StatisticalAggregate voltageReading = 7;
-    StatisticalAggregate temperatureReading = 8;
-    StatisticalAggregate loopTimer = 9;
-    IntHistogram loopTimerDistribution = 11;
+    CanErrorCounter canErrorCount = 6 [deprecated=true];
```

Comment:
Yeah, they will still be accessible (so datalogger parser can run over the fsgp data) but discouraged from use in new data. Not that nanopb generates deprecation warnings, sadly.